### PR TITLE
Add support for X509_load_cert_file

### PR DESCRIPTION
**Pull request by alexanderjordanbaker**

### CODE REVIEW

File: openssl/src/x509/store.rs [hunk 1]
```diff
@@ -176,6 +176,37 @@ impl X509LookupRef<HashDir> {
     }
 }
 
+/// Marker type corresponding to the [`X509_LOOKUP_file`] lookup method.
+///
+/// [`X509_LOOKUP_file`]: https://www.openssl.org/docs/man1.1.1/man3/X509_LOOKUP_file.html
+pub struct ASN1;
```
Comment (sfackler) [inline openssl/src/x509/store.rs:182]: Why is this called `ASN1` and not e.g. `File`?

Comment (alexanderjordanbaker) [reply openssl/src/x509/store.rs]: Thank you for pointing that out, that was a reference that I didn't replace with the appropriate name when creating the PR. I've updated the name to be `File` to match the `HashDir` struct above

File: openssl/src/x509/store.rs [hunk 2]
```diff
@@ -176,6 +176,37 @@ impl X509LookupRef<HashDir> {
...
+    pub fn file() -> &'static X509LookupMethodRef<File> {
+        unsafe { X509LookupMethodRef::from_ptr(ffi::X509_LOOKUP_file()) }
+    }
+}
+
+#[cfg(not(boringssl))]
+impl X509LookupRef<File> {
+    #[corresponds(X509_load_cert_file)]
+    /// Specifies a file from which certificates will be loaded
+    pub fn load_cert_file(&mut self, file: &str, file_type: SslFiletype) -> Result<(), ErrorStack> {
```
Comment (sfackler) [inline openssl/src/x509/store.rs:197]: The type of `file` should be `impl AsRef<Path>` like here: https://docs.rs/openssl/latest/src/openssl/ssl/mod.rs.html#902-912

**Comment by sfackler** [comment]
LGTM other than the method signature!

**Comment by sfackler** [comment]
Thanks!

**Pull request merged**